### DISCUSSION TITLE
part-ii/2011-132/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#410] [Question comments]
Part II 2011 Riemann Surfaces: Paper 3, Section II, G 

https://questions.tripos.org/part-ii/2011-132/

**Comment by eha1234** [comment]
For the last part, there are n+1 potential branch points at the nth roots of unity and infinity. I considered the monodromy group. The loops around the nth roots of unity cycle the r roots of unity in the same way, so correspond to the same r cycle. The loop around infinity is the composition of these, and splits into n 1 cycles (i.e. is the identity) iff r divides n.
